roles/tomcat/tasks/main.yml

```diff
@@ -39,7 +39,7 @@
 - name: Download Tomcat
   get_url:
     url: "{{ dependencies_url.tomcat }}"
-    checksum: sha512:{{ dependencies_url.tomcat_sha512_checksum }}
+    checksum: "sha512:{{ lookup('url', dependencies_url.tomcat_sha512_checksum_url).split(' ')[0] }}"
```

Comment:
```suggestion
    checksum: "sha512:{{ dependencies_url.tomcat_sha512_checksum_url }}"
```

Reply:
> this need to be as it is and it's probably the reason that make me change also the activemq one

the code bellow works so I don't know why it wouldn't on this repo... Can you please investigate?

```yaml
---
- hosts: localhost
  gather_facts: no
  connection: local
  tasks:
    - get_url:
        url: "{{ tc.url }}"
        dest: /tmp/toto
        checksum: "{{ tc.checksum }}"
      vars:
        tc:
          url: http://archive.apache.org/dist/tomcat/tomcat-9/v9.0.45/bin/apache-tomcat-9.0.45-deployer.tar.gz
          checksum: sha512:http://archive.apache.org/dist/tomcat/tomcat-9/v9.0.45/bin/apache-tomcat-9.0.45-deployer.tar.gz.sha512
```